src/services/tech-radar/client.ts

```diff
@@ -0,0 +1,16 @@
+interface TechRadarClient {
+  getTechRadarTechs(): Promise<RadarTech[]>;
+}
+
+interface RadarTech {
```

Comment:
This interface refers to the blip right? Perhaps a fitting name would be "Blip", or "Technology", cause `RadarTech` is much more broader.

Also I would suggest to move this specific type to the folder `/src/types`, that way the components can safely use it without direct accessing this client.

Reply:
RadarTech was more like in "a Radar Technology" but I agree as it does not get read well enough
We also have to consider practices so I'll go for `RadarBlip` if it sounds good